draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -375,9 +375,11 @@ congestion controller.
 
 For QUIC v1 connections, the attacker can examine offsets in STREAM frames to
 determine whether packet number skips are deliberate, so Q-bit signal provides
-little new information. However, an endpoint that implements {{DATAGRAM}} and
-uses a loss-based congestion controller MAY include deliberately skipped packet
-numbers in the current Q run.
+no new information (but it does save the attacker the need to remove packet
```

Comment:
I have to take back what I said earlier about removing packet protection:  The attacker _does_ need to remove it to get the packet number.  It is the packet payload that does not need to be decrypted in this case (to get at holes in STREAM frames).

Reply:
The attacker needs to remove _header_ protection.  It does _not_ need to remove _packet_ protection when Q bit signal is available.  So I believe the text here is correct.  (The terminology used in the transport draft is _packet_ protection, not _payload_ protection.)

Reply:
Ah, OK.